Dockerfile

```diff
@@ -0,0 +1,40 @@
+
+# TODO: make this default to govuk-ruby once it's being pushed somewhere public
+# (unless we decide to use Bitnami instead)
+ARG base_image=ruby:2.7.2
```

Comment:
Oh, an optional thing: consider trying `ruby:2.7.2-slim` to reduce the bloat a bit. Worked fine for me on a couple of earlier ones.